tdrs-backend/tdpservice/stts/serializers.py

```diff
@@ -23,18 +23,14 @@ def get_code(self, obj):
         return obj.code
 
 
-class STTUpdateSerializer(serializers.ModelSerializer):
-    """STT serializer."""
+class STTPrimaryKeyRelatedField(serializers.PrimaryKeyRelatedField):
+    """Accept STT ID only for updates but return full STT in response."""
 
-    class Meta:
-        """Metadata."""
-
-        model = STT
-        fields = ["id"]
-        extra_kwargs = {"id": {"read_only": False}}
+    queryset = STT.objects.all()
```

Comment:
Setting the queryset for this type of field ensures that users cannot submit an invalid STT. Under the previous setup an invalid STT ID could potentially create a new record.